composer.json

```diff
@@ -86,7 +86,8 @@
         "phpunit/phpunit": "9.5.2",
         "sebastian/phpcpd": "6.0.3",
         "squizlabs/php_codesniffer": "3.5.8",
-        "dmore/chrome-mink-driver": "^2.7"
+        "dmore/chrome-mink-driver": "^2.7",
+        "phpstan/phpstan": "^0.12.82"
```

Comment:
It might be worth moving this to the correct alphabetical position. I also like to pin specific versions, so if a new version of a tool is released and detects a new issue, it doesn't inexplicably break the build.

Reply:
It does make sense, I'll fix it